Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.21 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.21 as builder
```

Comment:
ubi9 is for 1.21 but filippe noticed some python versions with it. @spolti is it ok to go with ubi9 1.21?

Reply:
idk, it needs to be tested.
I only tested rest-proxy which required some tweaks. Not sure how things will go here..

Reply:
the latest version of the code builds successfully locally.

Reply:
nice, they have updated the ubi8 go-toolset

Reply:
nice timing :)